src/pages/about.js

```diff
@@ -3,10 +3,14 @@ import React from 'react';
 import { Flex, Heading } from '@chakra-ui/core';
 import Layout from '../components/Layout';
 import ContactCard from '../components/about/ContactCard';
+import { Helmet } from 'react-helmet';
 
 export default function About() {
   return (
     <Layout>
+      <Helmet htmlAttributes={{ lang: 'en' }}>
```

Comment:
@chrisdemars let's move this to the `SEO.js` component to keep things slim. This will eliminate the need for the repeated code on each page, since the `SEO.js` component is mounted on every page!

Reply:
@magnificode change made and pushed.